index.html

```diff
@@ -1,48 +1,88 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./css/style.css" />
-    <title>Awesome Books</title>
-  </head>
-  <body>
-    <nav>
-      <p>Awesome books</p>
-      <ul class="navlist">
-        <li><button id="list">List |</button></li>
-        <li><button id="addnew">Add new |</button></li>
-        <li><button id="contact">Contact</button></li>
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <script src="https://kit.fontawesome.com/e83d856101.js" crossorigin="anonymous"></script>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Awesomebooks</title>
+  <link rel="stylesheet" href="./index.css">
+  <script src="https://cdn.jsdelivr.net/npm/luxon@2.1.1/build/global/luxon.min.js"></script>
```

Comment:
Great job @AbdimulhinYussuf3675 🤝

- [ ] Kindly make sure the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) Package is added as a .js file to the project (in full or minified version). The module is imported `import { DateTime } from "luxon";`. See [Get the current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time)See [Get the current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time).

Reply:
@1GORDON I have imported in form of CDN, is it mandatory to have the file in full?